newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -292,7 +293,6 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
               label={<Trans>Hide the menu bar in the preview window</Trans>}
             />
           </Line>
-          {electron && (
```

Comment:
You should keep this line or use `<> ... </>`

![image](https://user-images.githubusercontent.com/1280130/119643885-34d61200-be14-11eb-8986-429ad865d2df.png)

Reply:
_Always make a test before_
Sorry ^^